app/views/splash/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<section class="name-container container">
+ <h2>Welcome to <span class="title">Budget App<span></h2>
+ <div class="d-flex gap-3 text-center">
+  <%= link_to "Sign In", new_user_session_path() %>
+  <p> | </p>
+  <%= link_to "Sign Up", new_user_registration_path() %>
+ </div>
+</section>
```

Comment:
- [ ] As per the requirements, you need to implement the layout, font-face and colors of your app with the given design.

given screens|your design
---|---
![image](https://github.com/jonas-45/budget-app/assets/91486791/a3704833-dd31-4039-aa94-806fcf2b32d2)![image](https://github.com/jonas-45/budget-app/assets/91486791/8ca5fb65-acc3-4367-aa4c-95158bfca328)|![image](https://github.com/jonas-45/budget-app/assets/91486791/b0936758-9e93-47a0-a90e-005cd7858c9a)![image](https://github.com/jonas-45/budget-app/assets/91486791/418f995f-f2f2-4c38-a1b7-15bc582f0325)